rocAL/include/augmentations/geometry_augmentations/node_crop_resize.h

```diff
@@ -31,15 +31,25 @@ class CropResizeNode : public CropNode {
     CropResizeNode() = delete;
     void init(float area, float aspect_ratio, float x_center_drift, float y_center_drift);
     void init(FloatParam *area, FloatParam *aspect_ratio, FloatParam *x_drift_factor, FloatParam *y_drift_factor);
+    void init(unsigned int crop_h, unsigned int crop_w, float x_drift, float y_drift,
+              RocalResizeInterpolationType interpolation_type = RocalResizeInterpolationType::ROCAL_LINEAR_INTERPOLATION);
     unsigned int get_dst_width() { return _outputs[0]->info().max_shape()[0]; }
     unsigned int get_dst_height() { return _outputs[0]->info().max_shape()[1]; }
-    std::shared_ptr<RocalRandomCropParam> get_crop_param() { return _crop_param; }
+    std::shared_ptr<CropParam> get_crop_param() {
+        if (_is_random_crop)
```

Comment:
please use ? : statement here

Reply:
Ternary operator throws an error due to returning different objects
```
In file included from /data/diffusion_experiments/rocAL/rocAL/include/meta_data/meta_node_crop_resize.h:30:                                                                                               
/data/diffusion_experiments/rocAL/rocAL/include/augmentations/geometry_augmentations/node_crop_resize.h:38:74: error: incompatible operand types ('shared_ptr<RocalRandomCropParam>' and 'shared_ptr<Rocal
CropParam>')                                                                                                                                                                                              
   38 |     std::shared_ptr<CropParam> get_crop_param() { return _is_random_crop ? _crop_param : _crop_fixed_param; }
```